front-end/src/ui-components/IdentityBadge.tsx

```diff
@@ -45,6 +45,7 @@ const IdentityBadge = ({ className, identity }: {className?: string, identity: D
 		{identity?.riot && <li><span className='desc'>riot:</span>{identity.riot}</li>}
 		{identity?.twitter && <li><span className='desc'>twitter:</span>{identity.twitter}</li>}
 		{identity?.web && <li><span className='desc'>web:</span>{identity.web}</li>}
+		{flags?.isCouncil && <li><span className='desc'>Council Member</span></li>}
```

Comment:
I think this is too light vsually.
As a user I want to see right away if a member that takes part in a conversation is from the council, without having to point my mouse over their name. Hence the suggestion in the issue "an additional icon" such as a crown (:crown:) or a badge next to the name, similar to the check for verified identities.

Reply:
A fun way would be something like
![image](https://user-images.githubusercontent.com/33178835/90008536-c8cb5880-dc9c-11ea-9f7f-0aa4b8ddef02.png)

Or even better (but also more work)
![image](https://user-images.githubusercontent.com/33178835/90008963-7d657a00-dc9d-11ea-9797-0dca41373eaf.png)

Reply:
Crown over identicon is hard to do. Added inplace of tick

Reply:
![Screenshot from 2020-08-12 18-14-26](https://user-images.githubusercontent.com/874046/90016554-caf6dc80-dcc7-11ea-9e1b-2fa8903c4078.png)

Reply:
The verification tick is as important as the crown since it shows whether this is an imposter or not, you can't just replace it.

Reply:
![Screenshot from 2020-08-12 18-30-15](https://user-images.githubusercontent.com/874046/90018166-232ede00-dcca-11ea-8dee-61335e1983d7.png)
What about both

Reply:
Yes, that's what I meant originally. It's not perfect but better than nothing.